runtime/dancebox/tests/common/mod.rs

```diff
@@ -30,54 +31,55 @@ use {
     sp_core::{Get, Pair},
     sp_runtime::{Digest, DigestItem},
     test_relay_sproof_builder::ParaHeaderSproofBuilder,
+    tp_consensus::runtime_decl_for_tanssi_authority_assignment_api::TanssiAuthorityAssignmentApi,
 };
 
 pub use dancebox_runtime::{
-    AccountId, Aura, Authorship, Balance, Balances, Initializer, ParachainInfo, Registrar, Runtime,
-    RuntimeCall, RuntimeEvent, Session, System,
+    AccountId, Balance, Balances, Initializer, ParachainInfo, Registrar, Runtime, RuntimeCall,
+    RuntimeEvent, Session, System,
 };
 
-pub fn run_to_session(n: u32, add_author: bool) {
+pub fn run_to_session(n: u32) {
     let block_number = dancebox_runtime::Period::get() * n;
-    run_to_block(block_number + 1, add_author);
```

Comment:
we could keep the `add_author`:

- include the nimbus log only when `add_author` is true 
- run the kickoff authorship validation inherent only when  `add_author` is true

In the future we might add better testing for this

Reply:
I think it caused other `expect` to fail :/

Reply:
ok nvm, we can move it later